lib/utils/link.js

```diff
@@ -43,6 +43,10 @@ function ensureFromIsNotSource (from, to, cb) {
   })
 }
 
+function unlinkAndIgnoreErrors (file, cb) {
+  fs.unlink(file, () => cb())
```

Comment:
`test/tap/unit-link.js` needs to be updated, since it mocks `fs`. It's failing with `fs.unlink is not a function` right now.